build.gradle

```diff
@@ -1441,6 +1441,7 @@ project(':connect:basic-auth-extension') {
   dependencies {
     compile project(':connect:api')
     compile libs.slf4jApi
+    compile libs.jerseyContainerServlet
```

Comment:
Is this necessary as a compile dependency for the API module? If so, then it seems like something might be in API when it should be in runtime.

Reply:
This is a dependency for connect:basic-auth-extension. It requires jersey implementation.

Reply:
Have made it testRuntime dependency